packages/react-charting/etc/react-charting.api.md

```diff
@@ -681,6 +681,7 @@ export interface IHorizontalBarChartProps {
     className?: string;
     culture?: string;
     data?: IChartProps[];
+    enableGradient?: boolean;
```

Comment:
This file will have more entries based on the new properties that you have added. This file will get auto generated as you execute yarn build inside charting package